geometry_msgs/Vector3 Mask
geometry_msgs/Vector3 DesiredRPY
---
geometry_msgs/Vector3 CurrentDesiredRPY
